bundles/org.eclipse.ui.ide/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %Plugin.name
 Bundle-SymbolicName: org.eclipse.ui.ide; singleton:=true
-Bundle-Version: 3.19.100.qualifier
+Bundle-Version: 3.19.200.qualifier
```

Comment:
Fail is:

```
 Error: 0.482 [ERROR] Failed to execute goal org.eclipse.tycho.extras:tycho-p2-extras-plugin:3.0.0-SNAPSHOT:compare-version-with-baselines (compare-attached-artifacts-with-release) on project org.eclipse.jface: Only qualifier changed for (org.eclipse.jface/3.27.0.v20220906-1112). Expected to have bigger x.y.z than what is available in baseline (3.27.0.v20220817-1444) -> [Help 1]
```

Am I supposed to bump some other plug-in here?

Reply:
Not really as part of this change. Can you please just make sure you change is properly rebased on top of master?

Reply:
> Not really as part of this change. Can you please just make sure you change is properly rebased on top of master?

See https://github.com/eclipse-platform/eclipse.platform.ui/pull/298#issuecomment-1239293153

Reply:
I will push a fix in https://github.com/eclipse-platform/eclipse.platform.ui/pull/318, which also was suffering from that issue